src/backends/glvisualize.jl

```diff
@@ -1382,7 +1382,7 @@ end
 
 
 function gl_heatmap(x,y,z, kw_args)
-    get!(kw_args, :color_norm, Vec2f0(extrema(z)))
+    get!(kw_args, :color_norm, Vec2f0(NaNMath.extrema(z)))
```

Comment:
@SimonDanisch I have replaced `minimum`, `mean`, `maximum`, `max` and `min` with versions from the NaNMath package that ignore `NaN`, to maintain the julia-0.5 behaviour of these functions. I have only done the changes where I believe the argument is likely to be a subtype of `AbstractFloat` or `AbstractArray{<:AbstractFloat}` as I believe this is the most likely place to find `NaN`s. Can you check that 1. all of the changes appear valid, and 2. that we don't need to identify NaN-compliant versions for other types? E.g. in line 1428 you take the `minimum` of an `AABB` object, but I'm guessing that will never be NaN?